### PR TITLE
Move workflows console flags to feature previews

### PR DESCRIPTION
**Pull request by cjllanwarne**
## Summary of changes:
Workflows tab and Find Workflows can now be enabled via the `#feature-preview` page:
image

This enables the workflows tab in Azure workspaces permanently, and without any need for config overrides in the javascript console:
image

### What
- Moves Workflows feature flags into the `#feature-preview` page (previously these were config overrides in the javascript console)

### Why
- To make it easier to opt-in to workflows features.
- To make the use of feature flags more standard

### Testing strategy
- Tested locally.

### CODE REVIEW

**Review by kpierre13 — APPROVED** [review]

**Review by dillydally414 — APPROVED** [review]

**Comment by dillydally414** [comment]
I think because workspaces team is marked as code owners you'll need a review from one of them to merge, maybe drop a message in the workspaces channel?

**Review by cahrens — APPROVED** [review]

**Review by aherbst-broad — APPROVED** [review]

**Pull request merged**
